.github/workflows/build.yml

```
@@ -114,26 +117,45 @@ jobs:
+          key: search-${{ matrix.platform.suffix }}-${{ hashFiles('./deps/RediSearch/src/version.h') }}
+          enableCrossOsArchive: true
+
+      - name: Cache libcurl ${{ matrix.platform.suffix }}
+        id: cache_libcurl
+        uses: actions/cache@v4
+        with:
+          path: /FalkorDB/bin/${{ matrix.platform.cache_path  }}-release/libcurl
+          key: libcurl-${{ matrix.platform.suffix  }}-${{ hashFiles('./deps/libcurl/RELEASE-NOTES') }}
+          enableCrossOsArchive: true
+
+      - name: Cache libcsv ${{ matrix.platform }}
```

actionlint (1.7.4)

object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type {build_test_image: bool; cache_path: string; machine_label: string; name: string; os: string; platform: string; suffix: string; test_dockerfile: string}

(expression)